larky/pom.xml

```diff
@@ -43,6 +43,7 @@
         <google.jimfs.version>1.2</google.jimfs.version>
         <google.re2j.version>1.6</google.re2j.version>
         <google.truth.version>1.1.3</google.truth.version>
+        <google.gson.version>2.9.1</google.gson.version>
```

Comment:
Why do you need this? We already have a json parser that we've built as part of internal starlark.

Reply:
I am not OK w/ adding another dependency tbh.

Reply:
agree, I didn't see we have one. Will use the internal one, thx